linker/Mono.Linker.Steps/MarkStep.cs

```diff
@@ -1136,6 +1182,11 @@ static internal bool IsPropertyMethod (MethodDefinition md)
 				(md.SemanticsAttributes & MethodSemanticsAttributes.Setter) != 0;
 		}
 
+		static internal bool IsPublicInstancePropertyMethod(MethodDefinition md)
```

Comment:
I don't understand the question. This is an internal method, not a property, just like IsPropertyMethod and IsEventMethod. What's your preference?